src/sr_od/app/sim_uni/sim_uni_app.py

```
@@ -78,26 +103,119 @@ def _check_times(self) -> OperationRoundResult:
+            kill_script_after_done=False,
+            kill_game_after_done=False,
+            notify_start=False,
+            notify_done=False,
+        )
+        BackToNormalWorldPlus(self.ctx).execute()
+
+        # 删除运行记录
+        plugin_run_result_path = os.path.join(plugin_path, 'logs', 'notif.txt')
+        if os.path.exists(plugin_run_result_path):
+            with open(plugin_run_result_path, 'w', encoding='utf-8') as file:
+                pass  # 不写入任何内容，仅清空
```

Ruff (0.13.1)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)